vulnerabilities/helpers.py

```diff
@@ -267,3 +267,81 @@ def _get_gh_response(gh_token, graphql_query):
     endpoint = "https://api.github.com/graphql"
     headers = {"Authorization": f"bearer {gh_token}"}
     return requests.post(endpoint, headers=headers, json=graphql_query).json()
+
+
+def get_reference_id(url: str):
+    """
+    Return the reference id from a URL
+    For example:
+    >>> get_reference_id("https://github.com/advisories/GHSA-c9hw-wf7x-jp9j")
+    'GHSA-c9hw-wf7x-jp9j'
+    """
+    url_parts = url.split("/")
```

Comment:
Using a partition is always more robust IMHO
```suggestion
    _url, _, ref_id = url.strip("/").rpartition("/")
```